### PR TITLE
Changes needed due to switch to new markdown in jekyll.

### PR DESCRIPTION
**Pull request by prachidamle**
Also support for layout,lang,version parameters
New description and schema files

Also added an option to generate empty description file

### CODE REVIEW

**Comment by prachidamle** [comment]
@deniseschannon these are all the changes we did for the new markdown.

**Comment by prachidamle** [comment]
@ibuildthecloud @cjellick can one of you merge this PR so that api-docs tool is updated with latest changes

File: scripts/gen-api-docs.sh [hunk 1]
```diff
@@ -3,11 +3,11 @@ set -e
 
 cd $(dirname $0)/../docs
 
-curl -s http://localhost:8080/v1/schemas?_role=project | jq . > ./input/schemas.json
+curl -s http://130.211.150.101:8080/v1/schemas?_role=project | jq . > ./input/schemas.json
```
Comment (cjellick) [inline scripts/gen-api-docs.sh]: Should this remain localhost?

Comment (prachidamle) [inline scripts/gen-api-docs.sh]: Yep, changed to localhost

**Comment by cjellick** [comment]
LGTM

**Comment by prachidamle** [comment]
Thanks @cjellick

**Pull request merged**